web/modules/custom/myroute/myroute.permissions.yml

```diff
@@ -0,0 +1,13 @@
+view myroute:
+  title: View my module.
+  description: View the my module module's content.
```

Comment:
Please encapsulate title and description values in single quotes.

Reply:
Done